.github/workflows/workflow.yml

```diff
@@ -198,7 +198,7 @@ jobs:
         env:
           # split tests across multiple machines
           CI_NODE_INDEX: ${{ matrix.machine }}
-          CI_NODE_TOTAL: 4
+          CI_NODE_TOTAL: 5
```

Comment:
Matches the number of machines from line 128 (i.e. `machine: ['0', '1', '2', '3', '4']`)